manual/src/tutorials/polymorphism.etex

```diff
@@ -343,7 +342,7 @@ Whereas the standard type "'a nested -> int" can be interpreted
 as ``let be a type variable "'a", then "depth" maps "'a nested" values
 to integers''. There are two major differences with these two type
 expressions. First, the explicit polymorphic annotation indicates to the
```

Comment:
```suggestion
inferred by OCaml might not be general enough to allow the definition of
some recursive functions, in particular for recursive functions acting on
```

Reply:
This is the wrong location for the change. The commit should be reverted and moved and applied at line 275 of polymorphism.etex .